### PR TITLE
v_summary_charts: fix typo of some arguments

### PR DESCRIPTION
**Pull request by HCY123902**
Commit message:
```
Fix the typo in summary chart methods

There seems to be typo in the parameter isMerged, with 
a character missing.

Handling the typo can enhance the consistency of the
parameter name with the naming convention.

Let's fix the typo in the parameter name.
```

### CODE REVIEW

**Review by Tejas2805 — APPROVED** [review]
LGTM!

File: frontend/src/static/js/v_summary_charts.js [hunk 1]
```diff
@@ -178,7 +178,7 @@ window.vSummaryCharts = {
         zLocation: this.getRepoLink(user),
         zSince: since,
         zUntil: until,
-        zIsMerge: isMerge,
+        zIsMerge: isMerged,
```
Comment (0blivious) [inline frontend/src/static/js/v_summary_charts.js:181]: shouldn't this also become sth like zIsMerged?

Comment (HCY123902) [reply frontend/src/static/js/v_summary_charts.js]: Thanks for pointing it out. I have changed it accordingly.

**Review by 0blivious — APPROVED** [review]

**Review by dcshzj — APPROVED** [review]

**Review by fzdy1914 — APPROVED** [review]

**Review by fzdy1914 — CHANGES_REQUESTED** [review]
image

**Review by fzdy1914 — APPROVED** [review]

**Pull request merged**